### DISCUSSION TITLE
zsh: no matches found: postcss@^7

### DISCUSSION THREAD

**Discussion by andrii1** [tailwindlabs/tailwindcss#3575] [Help] [answered]
Hello,

I'm trying to install Tailwind with React - simple create-react-app, following this guide https://tailwindcss.com/docs/guides/create-react-app

But getting the error
zsh: no matches found: postcss@^7
Any suggestions?

**Comment by adamwathan** [comment, marked as answer]
Try wrapping it in quotes, I think zsh treats ^ in a special way sometimes if it's not quoted.

**Reply by andrii1** [reply]
still the issue:

`npm install tailwindcss@npm:@tailwindcss/postcss7-compat @tailwindcss/postcss7-compat “postcss@^7” autoprefixer@^9
zsh: no matches found: “postcss@^7”`

Same issue with:
npm install “autoprefixer@^9”
zsh: no matches found: “autoprefixer@^9”

I'm on MacOS

**Reply by adamwathan** [reply]
Maybe this will help:

https://github.com/ohmyzsh/ohmyzsh/issues/449


Just googled "zsh caret no match found" 👍🏻

**Reply by andrii1** [reply]
haha it is solved! thanks a lot! I'm not English native speaker - didn't know how ^ is called lol... So, was googling all around zsh command 😅

**Reply by andrii1** [reply]
almost there...

after running npx tailwindcss init it doesn't create tailwind.config.js file:
  
   ```
@tailwindcss/postcss7-compat 2.0.3
  
   🚫 SyntaxError: Unexpected token } in JSON at position 589
    at JSON.parse (<anonymous>)
    at Object.isModule (/Users/andrii/coding-mentors/node_modules/@tailwindcss/postcss7-compat/lib/cli/utils.js:170:22)
    at /Users/andrii/coding-mentors/node_modules/@tailwindcss/postcss7-compat/lib/cli/commands/init.js:50:28
    at new Promise (<anonymous>)
    at Object.run (/Users/andrii/coding-mentors/node_modules/@tailwindcss/postcss7-compat/lib/cli/commands/init.js:48:10)
    at /Users/andrii/coding-mentors/node_modules/@tailwindcss/postcss7-compat/lib/cli/main.js:29:46
    at new Promise (<anonymous>)
    at run (/Users/andrii/coding-mentors/node_modules/@tailwindcss/postcss7-compat/lib/cli/main.js:25:10)
    at Object.<anonymous> (/Users/andrii/coding-mentors/node_modules/@tailwindcss/postcss7-compat/lib/cli.js:14:19)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
```

**Reply by andrii1** [reply]
Solved by manually creating tailwind.config.js file. Seems to work now 😁

**Reply by abinhho** [reply]
Add `\` should resolve the issue
```
npm install -D tailwindcss@npm:@tailwindcss/postcss7-compat @tailwindcss/postcss7-compat postcss@\^7 autoprefixer@\^9
```

**Reply by 97Fakhreddine** [reply]
@abinhho thanks dude that was helpful :)

**Reply by JoseAlz** [reply]
work!! in Manjaro KDE Plasma 5.22.4
`/Documentos/React_Projectos  npm install -D tailwindcss@npm:@tailwindcss/postcss7-compat @tailwindcss/postcss7-compat postcss@\^7 autoprefixer@\^9`

added 153 packages, and audited 154 packages in 34s

19 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

**Reply by Xenunxenunowski** [reply]
`npm install -D tailwindcss@npm:@tailwindcss/postcss7-compat @tailwindcss/postcss7-compat postcss@\^7 autoprefixer@\^9
`
solution works on Manjaro Gnome :D :+1:

**Comment by playmobilman** [comment]
@adamwathan Marry me man!, thank you!
